pkg/workspace/estimator/basicnodesestimator/estimator.go

```diff
@@ -50,12 +53,42 @@ func (e *BasicNodesEstimator) EstimateNodeCount(ctx context.Context, wObj *kaito
 	presetName := string(wObj.Inference.Preset.Name)
 	model := plugin.KaitoModelRegister.MustGet(presetName)
 
-	gpuConfig, err := utils.GetGPUConfigBySKU(wObj.Resource.InstanceType)
-	if err != nil {
-		return 0, fmt.Errorf("failed to get GPU config for instance type %s: %w", wObj.Resource.InstanceType, err)
-	}
-	if gpuConfig == nil {
-		return 0, fmt.Errorf("GPU config is nil for instance type %s", wObj.Resource.InstanceType)
+	// Import featuregates and consts for NAP check
+	var gpuConfig *sku.GPUConfig
+	var err error
+
+	// Check if NAP (Node Auto Provisioning) is enabled or disabled
+	napDisabled := featuregates.FeatureGates[consts.FeatureFlagDisableNodeAutoProvisioning]
```

Comment:
My question: I think since they are the same feature, so basicnodeestimator should have the same change as advanced, but seems like they are quite different, may I know the reason?